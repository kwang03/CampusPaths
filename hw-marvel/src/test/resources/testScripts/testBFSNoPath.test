#Tests BFS on finding a path that does not exist

LoadGraph myGraph twoCharsNoPath.csv

FindPath myGraph Char1 Char2