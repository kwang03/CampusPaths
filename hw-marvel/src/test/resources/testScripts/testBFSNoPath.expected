#Tests BFS on finding a path that does not exist

loaded graph myGraph

path from Char1 to Char2:
no path found